### ISSUE TITLE
Using git LFS to store the images result in empty image in report

### ISSUE THREAD

**Issue by dodozhang21** [garris/BackstopJS#1336]
After I ran the test using github action and have it commit the reports to the repo. I used github pages to load the report but the report has 1x1 pixel image for everything. I highly suspect this is related to the fact I use git LFS to store my images. 
The same test run is fine locally.

Any idea how to resolve this issue?
![image](https://user-images.githubusercontent.com/646064/124321203-29bc8300-db32-11eb-8f7f-75ddaebddfbb.png)

Thanks

**Comment by dodozhang21** [comment]
one idea is to go thru config.js changes and update every image using https://gist.github.com/fkraeutli/66fa741d9a8c2a6a238a01d17ed0edc5 :/

**Comment by garris** [comment]
Do the images get correctly saved?  Are you able to view the report if you download the repo?

**Comment by dodozhang21** [comment]
@garris yes the images are correctly saved. I can read the report when I download it. I've added a script to update image reference to the correct path for the server but it works half of the time

**Comment by edvardsniedre** [comment]
GitHub Pages doesn't support Git LFS: https://github.com/git-lfs/git-lfs/issues/4235
